loadtest/user/userentity/actions.go

```diff
@@ -142,6 +142,24 @@ func (ue *UserEntity) SearchPosts(teamId, terms string, isOrSearch bool) (*model
 	return postList, nil
 }
 
+func (ue *UserEntity) GetPostsForChannel(channelId string) ([]string, error) {
```

Comment:
This is the most controversial change and I want to explain why I did it this way:

All the Posts search returns PostList but in our case we only want to have store the Posts (by now). We're storing them in a slice indexed by PostId so there is no need to use the PostList. 

Also we don't want to tie our storage to the API model so there is no need to pass the PostList or even the Map.

As we made in other method, I built the ids for the posts and returned them in a plain slice

Reply:
I was thinking to add the `ChannelPosts` method but I didn't know if that would be too much logic for now or not. Chaging it...